build.gradle

```diff
@@ -390,12 +390,12 @@ project(':iceberg-pig') {
 project(':iceberg-spark-runtime') {
   apply plugin: 'com.github.johnrengelman.shadow'
 
-  tasks.build.dependsOn tasks.shadowJar
+  tasks.assemble.dependsOn tasks.shadowJar
   tasks.install.dependsOn tasks.shadowJar
   tasks.javadocJar.dependsOn tasks.shadowJar
 
   configurations {
-    compile {
+    compileOnly {
```

Comment:
By using `compile`, these dependencies were added to the artifact's transitive dependencies. I thought that was the right thing to do. Why should this not declare a dependency on, for example, slf4j?

Reply:
Okay, I just figured this one out. These are exclusions from dependencies that are now in the `compileOnly` scope. Sorry for the dumb question.